src/korean.js

```diff
@@ -59,6 +59,9 @@ class Korean {
     if (word[wordLength - 2] === '하') {
       conjugate = word.slice(0, wordLength - 2);
       return conjugate.concat('해');
+    } else if (word[wordLength - 2] === '앉' || word[wordLength - 2] === '솟' || word[wordLength - 2] === '닫') {
```

Comment:
@songz @minsooshin If anyone else wants to take a shot at this, feel free to go for it. I'm pretty busy this weekend. Hopefully, I'll see ya'll next weekend.